3-Authorization-II/2-call-api-b2c/SPA/public/index.html

```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

Comment:
```suggestion
    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```

let's remove this and the corresponding assets in public folder